/**:
  ros__parameters:
    # constraints param for normal driving
    normal:
      min_acc: -0.5 # min deceleration [m/ss]
      max_acc: 1.0 # max acceleration [m/ss]
      min_jerk: -0.5 # min jerk [m/sss]
      max_jerk: 1.0 # max jerk [m/sss]

    # constraints to be observed
    limit:
      min_acc: -2.5 # min deceleration limit [m/ss]
      max_acc: 1.0 # max acceleration limit [m/ss]
      min_jerk: -1.5 # min jerk limit [m/sss]
      max_jerk: 1.5 # max jerk limit [m/sss]

    # motion state constraints
    max_velocity: 20.0 # max velocity limit [m/s]
